google-cloud-bigquery-storage/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/storage/BigQueryStorageSettings.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.bigquery.storage
+
+import akka.actor.ClassicActorSystemProvider
+import akka.stream.alpakka.google.GoogleSettings
+import com.typesafe.config.Config
+
+final class BigQueryStorageSettings private (
+    val host: String,
+    val port: Int,
+    val rootCa: Option[String] = None,
+    val googleSettings: Option[GoogleSettings] = None
```

Comment:
`GoogleSettings` are handled separately so they can be shared across the modules, so this can be removed.